source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -413,6 +413,19 @@ absl::optional<SystemTime> SslSocket::expirationPeerCertificate() const {
   return Utility::getExpirationTime(*cert);
 }
 
+std::string SslSocket::sessionId() const {
+  SSL_SESSION* session = SSL_get_session(ssl_.get());
+  if (session == nullptr) {
+    return "";
+  }
+
+  unsigned int sessionIdLength = 0;
+  const uint8_t* sessionId = nullptr;
+
+  sessionId = SSL_SESSION_get_id(session, &sessionIdLength);
```

Comment:
style nits:
- variable names should be snake_case
- combine the declaration of sessionId (session_id) into one line.

Reply:
Ah, I should have caught that too! Thanks for pointing it out -- addressed.